Display the current todo list

Brian-Demon opened the pull request:
Added display functionality that allows the user to see the current todo list

Brian-Demon committed:
Display the current todo list

cadwallion commented:
So it looks like you added display in between the definition of `add` and the `desc` and `option` invocations that have to immediately precede the `add` function.  Thor requires that you invoke `desc` right before the method definition for it to detect the code it's going to run when it's tied together.

Additionally, there are warnings because you haven't added invocations of the `desc` method before `display` and `remove`.  Adding these `desc` calls to each of the commands should fix the code and remove the warnings.

Brian-Demon committed:
fixed issue with desc missing for methods

Brian-Demon commented:
Fixed

Brian-Demon committed:
Fixed display desc

Said "display [LIST]," now says "display"

Brian-Demon committed:
Fixed remove method

Fixed remove method from "Todo::Manager.new(config).add(item)" to "Todo::Manager.new(config).remove(item)"

Brian-Demon committed:
Updated remove method (TodoCLI)

Removed the Todo::Manager initialization in the remove method (TodoCLI) as the method does not exist yet in todo.rb (coming soon)

cadwallion reviewed (approved):
👍🏻
